docker-compose.yml

```diff
@@ -61,10 +61,13 @@ services:
       nofile:
         soft: 65536
         hard: 65536
-    # Memory limit for ES, as it tends to be a memory hoarder
-    # Set this value to an empty string to remove the limit
-    # https://docs.docker.com/compose/compose-file/compose-file-v2/#cpu-and-other-resources
-    mem_limit: ${ES_MEM_LIMIT:-4294967296} # 4 GiB in bytes
+    # Sets memory limit for ES, as it tends to be a memory hoarder.
+    # Remove this key to remove the limit.
+    # https://docs.docker.com/compose/compose-file/compose-file-v3/#deploy
+    deploy:
```

Comment:
The linked docs says `deploy` was added in the version 3 of the Compose file and this one is still in 2.x. It seems to me that it should be accompanied by the update of the file.